### PR TITLE
BCN- ELOI&FELICE

### PR DESCRIPTION
**Pull request by xxxelf**
@eloigil

### CODE REVIEW

**Comment by xxxelf** [comment]
@eloigil

**Comment by xxxelf** [comment]
@eloigil

File: starter-code/functions-and-arrays.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-}
+function maxOfTwoNumbers(first, second) {}
 
 var largest = maxOfTwoNumbers(2, 6);
 console.log(largest);
 
 // Finding Longest Word
-function findLongestWord (words) {
-
+function findLongestWord(words) {
```
Comment (tawebbcn) [inline starter-code/functions-and-arrays.js]: Good!

**Comment by tawebbcn** [comment]
/*
Good job!

My two most important improvements are:
- to avoid spaghetti code. you should always try to wrap block of codes into functions. Then you just call the function instead of duplicate the code block multiple times. You call averageNumbers(anotherNumberArray), averageNumbers(oneNumberArray), etc..
- try to always `return` something from a function, if you later want to console log it, you do it after.

example:
*/
var numbers = [2, 6, 9, 10, 7, 4, 1, 9];

// Calculate the Average
```
var result = numbers.reduce(function(accumulator, currentValue) {
  return accumulator + currentValue;
});
var average = result / numbers.length;
var numbers = [2, 6, 9, 10, 7, 4, 1, 9];
console.log(average);
```

// to ->

```
function averageNumbers(array) {
  var result = array.reduce(function(accumulator, currentValue) {
    return accumulator + currentValue;
  });
  return result / array.length;
}

var average = averageNumbers(numbers);
console.log(average);

```

// Isak

**Pull request closed without merging**